css/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Why do you need this css file?

Reply:
In order to display my stylings in the same way on all the browsers I used this file. Wouldn't you recommend it?